prow/cluster/monitoring/mixins/prometheus/hook_alert.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'abnormal webhook behaviors',
+        rules: [
+          {
+            alert: 'no-webhook-calls',
+            // utc time of 9a-5p PDT: with some offset on day_off_week
+            expr: |||
+              (sum(increase(prow_webhook_counter[1m])) == 0 or absent(prow_webhook_counter))
+              and ( (day_of_week() > 0) and (day_of_week() < 6) and ((hour() >= 18) or (hour() <= 2))
+            |||,
+            'for': '5m',
```

Comment:
I have no feel for whether this is a reasonable threshold, how often have we crossed this in the past?

Reply:
If I recall, two github silences period in the last month.
Oh, another source of the alerts ... for bugs (knock, knock, knock ...) in our prometheus api, we do not have sample published.

Reply:
Basically never but let me run the query and post it

Reply:
![Screenshot from 2019-07-24 15-22-27](https://user-images.githubusercontent.com/7328370/61832714-dd854600-ae26-11e9-883e-430fde7a414f.png)
@spiffxp looks like it would have fired a couple times in the last two months, but all of those times were when there was a `hook` outage or the metrics were otherwise unavailable -- there are zero cases where this would have fired due to inactivity during working hours